Mainframe3270/x3270.py

```diff
@@ -81,51 +81,44 @@ def open_connection(
             | Open Connection | Hostname |
             | Open Connection | Hostname | LU=LUname |
             | Open Connection | Hostname | port=992 |
-            | ${extra_args}   | Create List | -accepthostname | myhost.com | -cafile | ${CURDIR}/cafile.crt |
-            } append to list  | ${extra_args} | -port | 992 |
+            | @{extra_args}   | Create List | -accepthostname | myhost.com | -cafile | ${CURDIR}/cafile.crt |
+            | Append To List  | ${extra_args} | -port | 992 |
             | Open Connection | Hostname | extra_args=${extra_args} |
             | Open Connection | Hostname | extra_args=${CURDIR}/argfile.txt |
         """
-        self.host = host
```

Comment:
I noticed that this self.host, self.lu and self.port was only used in this function. Since I do not anticipate their usage elsewhere, I removed them to make the code easier to read